libraries/engage/product/components/UnitQuantityPicker/UnitQuantityPicker.jsx

```diff
@@ -0,0 +1,181 @@
+import React, { useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'glamor';
+import classNames from 'classnames';
+import { themeConfig } from '@shopgate/engage';
+import { RippleButton, QuantityInput } from '@shopgate/engage/components';
+import { useWidgetSettings } from '@shopgate/engage/core';
+
+const { variables, colors } = themeConfig;
+
+const styles = {
+  root: css({
+    display: 'flex',
+    flexDirection: 'row',
+  }).toString(),
+  input: color => css({
+    padding: `0 ${variables.gap.small}px`,
+    textAlign: 'center',
+    flex: 1,
+    fontSize: 15,
+    height: 28,
+    width: '100%',
+    backgroundColor: colors.shade8,
+    color,
+  }).toString(),
+  inputWrapper: css({
+    width: '100%',
+  }),
+  button: bgColor => css({
```

Comment:
i forgot about something....
once you can change the bgColor, you should also be able to change the text color.

So we need 4 configs ins total:
input: textColor and bgColor
buttons: textColor and bgColor